core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -1215,7 +1216,15 @@ object AutoTuner extends Logging {
       val reader = new BufferedReader(new InputStreamReader(fsIs))
       val fileContent = Stream.continually(reader.readLine()).takeWhile(_ != null).mkString("\n")
       loadClusterPropertiesFromContent(fileContent)
-    } finally {
+    }
+    catch {
```

Comment:
nit, these should go on previous line 
" } catch {"
"} finally {"

Reply:
Updated